ni/measurements/metadata/v1/registration.schema.json

```diff
@@ -0,0 +1,407 @@
+        },
+        "link": {
+          "type": "string",
+          "description": "A URI to a resource that further describes this test adapter"
+        },
+        "schemaId": {
+          "type": "string",
+          "description": "The unique identifier of the schema that applies to this instance's extensions"
+        },
+        "extensions": {
+          "$ref": "#/definitions/Extensions"
+        }
+      }
+    },
+    "Test": {
+      "type": "object",
+      "description": "Represents the metadata of a test definition",
+      "additionalProperties": false,
+      "properties": {
+        "alias": {
+          "type": "string",
+          "description": "Optional alias identifier for referencing this test from other entities"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the test"
+        },
+        "description": {
+          "type": "string",
+          "description": "Explanation of what the test does"
+        },
+        "link": {
+          "type": "string",
+          "description": "A URI to a resource that further describes this test"
+        },
+        "schemaId": {
+          "type": "string",
+          "description": "The unique identifier of the schema that applies to this instance's extensions"
+        },
+        "extensions": {
+          "$ref": "#/definitions/Extensions"
+        }
+      }
+    },
+    "TestStation": {
+      "type": "object",
+      "description": "Represents the metadata of a test station where tests are executed",
+      "additionalProperties": false,
+      "properties": {
+        "alias": {
+          "type": "string",
+          "description": "Optional alias identifier for referencing this test station from other entities"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the test station"
+        },
+        "assetIdentifier": {
+          "type": "string",
+          "description": "The asset identifier for tracking and inventory purposes"
+        },
+        "link": {
+          "type": "string",
+          "description": "A URI to a resource that further describes this test station"
+        },
+        "schemaId": {
+          "type": "string",
+          "description": "The unique identifier of the schema that applies to this instance's extensions"
+        },
+        "extensions": {
+          "$ref": "#/definitions/Extensions"
+        }
+      }
+    },
+    "Uut": {
+      "type": "object",
+      "description": "Represents the metadata of a Unit Under Test (UUT) model or type",
+      "additionalProperties": false,
+      "properties": {
+        "alias": {
+          "type": "string",
+          "description": "Optional alias identifier for referencing this UUT from other entities"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the UUT model"
+        },
```

Comment:
Rename to modelName to match field in the [proto file](https://github.com/ni/ni-apis/blob/users/nbeer/metadata-schema/ni/measurements/metadata/v1/metadata_store.proto#L272).

Reply:
Will do - also requires a change to the backend service.

Reply:
updated.